tests/data_out/data_out_base_vtu_04.with_zlib=on.output

```diff
@@ -9,7 +9,7 @@
+AQAAAAASAAAAEgAAVwUAAA==eNqdV0FoXVUQ/VW0FhWhSQutWYSaX1SQVqrQFv67tlQR7EJa/kLUSjeKdlNcuFEkVMFGTEGkuy4ayXeRgBK0gtI3E4ggLiQWiUKkglQ3hUDQUqiKOHPfzL3n36z+z+bOyxvSc17PmTun1ZoMLfs5ef9bYXX50Urrr86dDpfGPov1z5+/qD2xb37utdS/Y8cr4b3/dsWelTMvhH/evRjrvQefCfNz12Pd7XZT/5Xtx8Ljj4zG36+/8XQ4vPZhrKcW94dud6Vqup5I/VOjnXDt+Tvi7+8+9Vg4230n1r89tVt6FjdgW9p5JGyenoi1nvpsfypYXx+2fUv7woWftsff66nPDcfrlfaV2D4+9UDYO76l+Tty6nPDcaXSvhLb1tGR8P6PN2Otpz43cBarpm9wvvNzD6f+Q1sfDLu2rXW03v3SeLjw5qux5+D3I8Jxzr7/ttR/676R8Pbry7H/0PF7wkNHj8aej45vEo7nK/9WWQ//VqvLC51GD+vVpbE99s2vyjlZDcMXsZ25fFv44uVfYq2nPjcc5yrtK7Fd++736uotjrWe+txwPF9pX4ltaec31ebp2arRw2ylzw2eyarp69eG6l11r7X6QP3QtPzQbplPUBuqd9W91uoD9YPWnx74q+0+QW2o3lX3WqsP1A/RR/un2tkn+Vvp/7/qIHIXXag+Mh7XzXrCJpjJMQsXci7yw87x19X1hE0wk2MWLuRchCM5x4W7ziZsgpkcs3Ah5yIcyTkiNsFMjlm4kHMRjpT1Pxhf1IbqXXWvtfpA/aD1vR/UbfcJakP1rrrXWn2gftD66z8+aWefZG2o3lX3zbdd6Kgfmp5n29kng/F97sblhE0w145ZuJBzEY7kHI992UvYBHPtmIULORfhSM6x1TqdsAnm2jELF3IuwpFA/0kbMjPZZ6ac7LNU3nOe/1kbMjPZZ6ac7LNU3nOe/1kbMjPZZ6ac7LNU3nOe//lbyQwhnyFyss8Wec8w/xM20Tv7nSU+YL/LhK/2tEtsonf2O0t8wH6XiT9SP2ITvbPfWeID9rtM/JH6EZv8/7PPcNEF+2wXvaT+QfmiNmRmks9MOdlnqbznPP+zNmRmks9MOclnqbznPP+zNmRmks9MOclnqbwnmP8D8UVsonf2O0t8wH6XiT/475k/J0psonf2O0t8wH6XiT/4zhPTEyU20Tv5nSU+IL/LVPPSM1FqwzRDoCUCjXGpDdMMgZYINEalNkwzBFoi0BiV38q+IcG3JfjmG7CZZxm8zOBxLrGZZxm8zOBxLrGZZxm8zOBxLrGZhgm0zaB5HoYvasM0U4OWCDRGpTZMMzVoiUBjVGrDNFODlgg0RsPwRWzmWQIvM3icS2zmWQIvE3icS2zmWQIvE3icSm3YzsCwSxj+yZ75pE8btjMw7BKx3jPT6ZlP+rRhOwPDLhHrKycO98wnfd/K7lCGu9X13DPdyM94wmY7G8Mux7DjxXqmPpKw2c7GsMsx7Hix3rL2ZMJmOxvDLsew423AZjsMw27DsPMMxRe1YTsDwy6R+tUnpTZsZ2DYJVK/+qTUhu0MBLsEZTytofiOfbspYbOdjWGXY9jxYn3g3O0Jm+1sDLscw45n/9bFWcdmOxvBLkew41GpDcuMDFmSIWNu0IZlRoYsyZAxqdSGZUaGLMmQMan8VpahGLIVQeaiEptldoIs73xTpkNsltkJsjyVGR+xWWYnyPJUZnzEZhmWINtSmXkH5YvasMzIkCUJMiaV2rDMSJAlCTLmBm1YZiTIkgQZk4bhi9gssxNk+brM+IjNMjtBlq/LjI/YLLMTZPm6zPj/A1hg9+k=
```

Comment:
This is funny -- why does your patch change a .vtu file?

Reply:
The test passes without modifications to the output file when `deal.II` is configured without `ZLIB` support. Hence, there should only be a small difference that results in a different output here.

Reply:
The fact that this changes by going from `a/b` to `a * (1/b)` shows why compilers do not do this optimization automatically: It changes in floating point arithmetic (even though only slightly in the last one or two bits for most reasonable numbers).